pkg/kubelet/util/sliceutils/sliceutils.go

```diff
@@ -32,7 +32,7 @@ func StringInSlice(s string, list []string) bool {
 	return false
 }
 
-// PodsByCreationTime makes an array of pods sortable by their creation
```

Comment:
please revert the changes in this file, we don't want to wait for a kubelet review.